pyspark/bigdl/models/inception/inception.py

```diff
@@ -0,0 +1,356 @@
+    parser.add_option("--maxLr", type=float, dest="maxLr", default=0.0, help="max Lr after warm up")
+    parser.add_option("-b", "--batchSize", type=int, dest="batchSize", help="batch size")
+    parser.add_option("--classNum", type=int, dest="classNum", default=1000, help="class number")
+    parser.add_option("--weightDecay", type=float, dest="weightDecay", default=0.0001, help="weight decay")
+    parser.add_option("--checkpointIteration", type=int, dest="checkpointIteration", default=620,
+                      help="checkpoint interval of iterations")
+    parser.add_option("--gradientMin", type=float, dest="gradientMin", default=0.0, help="min gradient clipping by")
+    parser.add_option("--gradientMax", type=float, dest="gradientMax", default=0.0, help="max gradient clipping by")
+    parser.add_option("--gradientL2NormThreshold", type=float, dest="gradientL2NormThreshold", default=0.0, help="gradient L2-Norm threshold")
+
+    return parser
+
+
+if __name__ == "__main__":
+    # parse options
+    parser = config_option_parser()
+    (options, args) = parser.parse_args(sys.argv)
+    if not options.learningRate:
+        parser.error("-l --learningRate is a mandatory opt")
+    if not options.batchSize:
+        parser.error("-b --batchSize is a mandatory opt")
+
+    # init
+    sparkConf = create_spark_conf().setAppName("inception v1")
+    sc = get_spark_context(sparkConf)
+    redire_spark_logs()
+    show_bigdl_info_logs()
+    init_engine()
+
+    image_size = 224  # create dataset
+    train_transformer = Pipeline([PixelBytesToMat(),
+                                  RandomCrop(image_size, image_size),
+                                  RandomTransformer(HFlip(), 0.5),
+                                  ChannelNormalize(0.485, 0.456, 0.406, 0.229, 0.224, 0.225),
+                                  MatToTensor(to_rgb=True),
+                                  ImageFrameToSample(input_keys=["imageTensor"], target_keys=["label"])
+                                  ])
+    raw_train_data = get_inception_data(options.folder, sc, "train")
+    train_data = DataSet.image_frame(raw_train_data).transform(train_transformer)
+
+    val_transformer = Pipeline([PixelBytesToMat(),
+                                CenterCrop(image_size, image_size),
+                                RandomTransformer(HFlip(), 0.5),
+                                ChannelNormalize(0.485, 0.456, 0.406, 0.229, 0.224, 0.225),
+                                MatToTensor(to_rgb=True),
+                                  ImageFrameToSample(input_keys=["imageTensor"], target_keys=["label"])
+                                ])
+    raw_val_data = get_inception_data(options.folder, sc, "val")
+    val_data = DataSet.image_frame(raw_val_data).transform(val_transformer)
+
+    # build model
+    if options.model != "":
+        inception_model = Model.load(options.model)
```

Comment:
Uer Model.loadModel(...) to use bigdl format model ?

Reply:
Here we load model checkpoint, which is saved by model.save not model.saveModule. So use Model.load API.